_episodes/03-create.md

```diff
@@ -215,15 +214,15 @@ draft.txt
 > 3.  When might you want to create a file this way?
 >
 > > ## Solution
-> > 1.  The touch command generates a new file called 'my_file.txt' in
+> > 1.  The `touch` command generates a new file called `my_file.txt` in
```

Comment:
`touch` creates a new file in the current directory, which isn't necessarily the home directory.